docs/mobile/exercises/README.md

```diff
@@ -169,17 +167,17 @@ contains a short string of text that is displayed in the footer of the
 table. The most common use for the table message is to indicate row truncation,
 because tables should not be displayed with more than 5 rows as a general
 rule. If you *are* truncating the result, the `message` field is mandatory,
-otherwise it is not.
+otherwise it is not. The standard format for indicating truncation is the one
+featured in the example above: `"Showing n out of N rows"`.
 
 ## image
 
 ```yaml
-image: my-cool-plot
+image: assets/images/my-cool-plot.png
 ```
 
-You can put image assets into your exercises! In this example, `my-cool-plot` is
-a reference to an image asset specified in the [course
-manifest](../repo-structure.md).
+You can insert an image block into your exercise simply by specifying the path
+to a PNG file. This is useful for things like teaching about plotting.
```

Comment:
Should we say something about not having too large images? As people will have to download these on their phone, possible over their cellular connection?

Reply:
Can do.